tests/OSPSuite.Core.Tests/Core/CurveChartSpecs.cs

```diff
@@ -50,27 +67,31 @@ public void the_relative_legend_indicies_should_be_adjusted()
 
    public class When_adding_new_Y_axes : concern_for_CurveChart
    {
-      private Curve _curve;
+      private Axis _y2Axis;
 
       protected override void Context()
       {
          base.Context();
          sut.DefaultYAxisScaling = Scalings.Linear;
-
-         _curve = A.Fake<Curve>();
-
-         sut.AddCurve(_curve);
       }
 
       protected override void Because()
       {
-         sut.SetyData(_curve, A.Fake<DataColumn>(), A.Fake<IDimensionFactory>());
+         _y2Axis = sut.AddNewAxisFor(AxisTypes.Y2);
       }
 
       [Observation]
       public void the_default_y_axis_scaling_should_be_used_for_the_new_axis()
       {
-         sut.Axes.First().Scaling.ShouldBeEqualTo(Scalings.Linear);
```

Comment:
This was wrong. First returns X Axis in that case but the test should have tested Y axis scaling....